scripts/build.mjs

```diff
@@ -24,15 +36,6 @@ async function main() {
     sourcemap: false,
   });
 
-  // Remove the types file from the dist-src folder
-  const typeFiles = await glob([
-    "./pkg/dist-src/**/types.js.map",
-    "./pkg/dist-src/**/types.js",
-  ]);
-  for (const typeFile of typeFiles) {
-    await rm(typeFile);
-  }
-
```

Comment:
Why was this removed? Whenever we have a `types.ts` file it would create an empty `types.js` file which is useless